src/core/mempool.rs

```diff
@@ -1360,16 +1640,231 @@ impl MemPoolDB {
             }
         }
     }
+
+    /// Get the bloom filter that represents the set of recent transactions we have
+    pub fn get_txid_bloom_filter(&self) -> Result<BloomFilter<BloomNodeHasher>, db_error> {
+        self.bloom_counter.to_bloom_filter(&self.conn())
+    }
+
+    /// Find maximum height represented in the mempool
+    fn get_max_height(conn: &DBConn) -> Result<Option<u64>, db_error> {
+        let sql = "SELECT 1 FROM mempool WHERE height >= 0";
+        let count = query_rows::<i64, _>(conn, sql, NO_PARAMS)?.len();
+        if count == 0 {
+            Ok(None)
+        } else {
+            let sql = "SELECT MAX(height) FROM mempool";
+            Ok(Some(query_int(conn, sql, NO_PARAMS)? as u64))
+        }
+    }
+
+    /// Get the transaction ID list that represents the set of transactions that are represented in
+    /// the bloom counter.
+    pub fn get_bloom_txids(&self) -> Result<Vec<Txid>, db_error> {
+        let max_height = match MemPoolDB::get_max_height(&self.conn())? {
+            Some(h) => h,
+            None => {
+                // mempool is empty
+                return Ok(vec![]);
+            }
+        };
+        let min_height = max_height.saturating_sub(BLOOM_COUNTER_DEPTH as u64);
+        let sql = "SELECT mempool.txid FROM mempool WHERE height > ?1 AND height <= ?2 AND NOT EXISTS (SELECT 1 FROM removed_txids WHERE txid = mempool.txid)";
+        let args: &[&dyn ToSql] = &[&u64_to_sql(min_height)?, &u64_to_sql(max_height)?];
+        query_rows(&self.conn(), sql, args)
+    }
+
+    /// Get the transaction tag list that represents the set of recent transactions we have.
+    /// Generate them with our node-local seed so that our txtag list is different from anyone
+    /// else's, w.h.p.
```

Comment:
`w.h.p.` = ?

Reply:
With high probability.  My math professors used it a lot

Reply:
Ah, well, I think it's probably uncommon enough to warrant expansion in the comment!